src/pages/about.js

```diff
@@ -36,80 +36,83 @@ const GitHubIconContainer = styled.div`
 const About = ({ location, sidebarContent }) => {
   return (
     <Page sidebarContent={sidebarContent} location={location}>
-      <PageHeader>Victory: Charting for React and React Native</PageHeader>
-      <GitHubIconContainer>
-        {/*
-         * TODO: Customize these buttons
-         * https://github.com/FormidableLabs/formidable-landers/issues/175
-         */}
-        <iframe
-          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=star&count=true&size=large"
-          frameBorder="0"
-          scrolling="0"
-          width="160px"
-          height="30px"
-        >
-          &nbsp;
-        </iframe>
-        <iframe
-          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=watch&count=true&size=large&v=2"
-          frameBorder="0"
-          scrolling="0"
-          width="160px"
-          height="30px"
-        >
-          &nbsp;
-        </iframe>
-        <iframe
-          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=fork&count=true&size=large"
-          frameBorder="0"
-          scrolling="0"
-          width="158px"
-          height="30px"
-        >
-          &nbsp;
-        </iframe>
-      </GitHubIconContainer>
-      <p>
-        Victory is a set of modular charting components for React and React
-        Native. Victory makes it easy to get started without sacrificing
-        flexibility. Create one of a kind data visualizations with fully
-        customizable styles and behaviors. Victory uses the same API for web and
-        React Native applications for easy cross-platform charting.
-      </p>
-      <p>
-        Victory is helmed by Formidable’s{" "}
-        <a href="https://github.com/boygirl"> Lauren Eastridge</a>.
-      </p>
-      <p>
-        <a href="https://github.com/FormidableLabs/victory/graphs/contributors">
-          See Victory Contributors
-        </a>
-      </p>
-
-      <Divider />
-
-      <Section>
-        <Subheader id="showcase">Victory in Use</Subheader>
+      <div>
```

Comment:
Only change in this file is wrapping with a `<div>` for consistency in how gutter is getting applied within the page.js `article`